src/pages/ManageVaccineStockPage.js

```diff
@@ -196,12 +196,10 @@ export class ManageVaccineStockPage extends React.Component {
   // be available.
   componentDidMount = () => {
     const { database, initialLocation } = this.props;
-    const fridges = database.objects('Location').filter(location => location.isFridge);
-    fridges.unshift({ description: LOCALIZATION.misc.allLocations });
+    const fridges = database.objects('Fridge');
+    fridges.slice().unshift({ description: LOCALIZATION.misc.allLocations });
```

Comment:
this.LOCATION_FILTERS = fridges.slice().uns...

Also suggest adding 'no fridges filter' quite useful to know which items are not assigned to fridge

Reply:
c32bcad2ada67ca870dbcd363f8865f35b718db0

7c6be3ae39256e7e65e2738bc90c5cfbec5bcca9